Podcast/UserDetailViewController.swift

```diff
@@ -19,6 +19,7 @@ final class UserDetailViewController: ViewController {
     
     let recastsHeaderViewTag = 2112
     let subscriptionsHeaderViewTag = 3110
+    let navBarHeight: CGFloat = 44
```

Comment:
let's make navBarHeight have a getter 
`get {
return navigationController?.navigationBar.frame.height ?? navBarHeight
}` so that we don't need to do the ?? everywhere but also have the correct height

Reply:
I'm actually going to remove that instance of navigationController?.navigationBar.frame.height because everything else is calculated to 44 anyway, so if they ever changed the height (like with iOS 11) it's going to be weird.